authorize_test.go

```diff
@@ -33,6 +34,16 @@ func TestIsAuthorized(t *testing.T) {
 			Want:      true,
 			DiagErr:   0,
 		},
+		{
+			Name:      "nil-entity-map",
+			Policy:    `permit(principal,action,resource);`,
+			Principal: cuzco,
+			Action:    dropTable,
+			Resource:  cedar.NewEntityUID("table", "whatever"),
+			Context:   cedar.Record{},
+			Want:      true,
+			DiagErr:   0,
+		},
```

Comment:
Can you explicitly initialize `Entities` to `nil` here?

Also, I guess I would name the test `nil-entity-getter`.

Reply:
Yes, done here: https://github.com/cedar-policy/cedar-go/pull/77/commits/9074d5b00f44be4346c13ac8b7270a09b06c541a.